docs/cli_tutorial/taker.md

```diff
@@ -0,0 +1,194 @@
+# Taker-Cli Tutorial
+
+## What is the Taker in Coinswap?
+
+A **taker** initiates a coinswap transaction, seeking to exchange their coins with the assistance of **makers**, without losing control of their funds.
+
+## Prerequisites
+
+You will need to have `bitcoind` set up before proceeding with the steps below.
+
+### Start Bitcoind
+
+Make sure you start your `bitcoind` daemon to work with the `taker-cli` by running the following command:
+
+```bash
+bitcoind -daemon
+```
+
+## Create a Wallet
+
+To perform basic operations such as generating new blocks, you'll need a wallet. Let's create a wallet named `default`.
+
+```bash
+bitcoin-cli createwallet "default"
+```
+
+## Send Funds to the `default` Wallet by Generating Blocks
+
+### Step 1: Get a New Address
+
+First, get a new address from the `default` wallet.
+
+```bash
+bitcoin-cli getnewaddress
+```
+
+### Step 2: Generate 101 Blocks and Send Funds
+
+Generate 101 blocks and send the funds to the address generated in Step 1.
+
+```bash
+bitcoin-cli generatetoaddress 101 <address>
```

Comment:
This will only work in regtest. Just mention about running bitcoind in regtest mode.